test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightPropertySerializerTests.cs

```diff
@@ -577,15 +579,55 @@ public void WritingPropertyShouldWriteCollectionResourceInstanceAnnotation()
                 result);
         }
 
+#if NETCOREAPP3_1_OR_GREATER
+        [Fact]
+        public void WritingJsonElementProperties_ShouldSerializeJsonInput()
+        {
+            string jsonInputString = "{\"foo\":\"bar\"}";
+            var jsonDocument = System.Text.Json.JsonDocument.Parse(jsonInputString);

```

Comment:
Why not import `System.Text.Json` namespace?

Reply:
Initially I thought I'd only use it in this single statement and didn't feel the need to add an import. But I think I should have. Let me add the using statement.